android/src/main/java/com/brentvatne/react/ReactVideoView.java

```diff
@@ -565,7 +572,10 @@ public void run() {
         }
 
         // Select track (so we can use it to listen to timed meta data updates)
-        mp.selectTrack(0);
+        try{
```

Comment:
Maybe you should add a comment why this try catch is there for future contributors

Reply:
for avoiding crash when drag the video progress on some old android phone. It will throw error code -1010, this error code was define in MediaPlayer class as below:
/** Bitstream is conforming to the related coding standard or file spec, but
     * the media framework does not support the feature. */
    public static final int MEDIA_ERROR_UNSUPPORTED = -1010;

Reply:
@cobarx What do you think? IMHO this should be added as a code comment to not confuse contributors in the future!